server.test.js

```diff
@@ -25,21 +29,122 @@ describe('Games', () => {
   });
   // declare some global variables for use of testing
   // hint - these wont be constants because you'll need to override them.
+
+
   beforeEach(done => {
     // write a beforeEach hook that will populate your test DB with data
     // each time this hook runs, you should save a document to your db
     // by saving the document you'll be able to use it in each of your `it` blocks
+    let tempTitle = null;
+    let tempId = null;
```

Comment:
`tempId` should be declared outside of `beforeEach`'s scope so your other tests can access it as well.